app/lib/theme/widgets/ReadOnlySubmissionWidget.tsx

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { WidgetProps } from '@rjsf/core';
+import styled from 'styled-components';
+import a from '../../../formSchema/schema';
+
+const StyledContainer = styled('div')`
+  margin-bottom: 8px;
+`;
+
+const StyledValue = styled('div')`
+  margin-top: 12px;
+  margin-bottom: 4px;
+  padding: 0.6em 0;
+  white-space: nowrap;
+`;
+
+const StyledError = styled('div')`
+  color: #e71f1f;
+  font-weight: 700;
+`;
+
+const StyledLink = styled(Link)`
+  color: #e71f1f;
+`;
+
+const ReadOnlySubmissionWidget: React.FC<WidgetProps> = ({
+  options,
+  value,
+}) => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const pageNumber =
+    Object.keys(a.properties).indexOf('organizationProfile') + 1;
+
+  const submissionCompletedFor =
+    options?.['field-name'] === 'submissionCompletedFor';
+
+  return (
+    <StyledContainer>
+      <StyledValue>
+        {value}
+        {submissionCompletedFor && !value && (
+          <StyledError>
+            No legal organization name was provided. Please return to the{' '}
+            <StyledLink href={`/form/${id}/${pageNumber}`}>
+              Organization Profile
+            </StyledLink>{' '}
+            page and enter one.
+          </StyledError>
```

Comment:
This pattern of having field-specific branching behaviour within widgets is at odds with having a declarative uiSchema. If we need specific behaviour for a single field, we should either create a component for it, or find a generic way to have this custom behaviour. 

Can the [custom validation](https://react-jsonschema-form.readthedocs.io/en/docs/usage/validation/#custom-validation-rules) add some JSX as an error?
A generic solution could be a `customError` uiSchema option that would be handled by our field template.

If we do need a custom component, in this case it should be a custom field, not a widget. Widgets should only concern themselves with rendering an input/value.